### PR TITLE
feat: react component template

### PR DESCRIPTION
**Pull request by debs-obrien**
## Proposed Changes

@davidfirst need to test locally to see if it works but not sure how to so let me know or merge and I will test it then

### CODE REVIEW

**Comment by debs-obrien** [comment]
working perfect locally but missing something to catch the index file when using namespaces. 
bit create react-component ui/card will put everything in a card folder but index.ts will be in the ui folder

File: .bitmap [hunk 1]
```diff
@@ -631,6 +631,12 @@
         "mainFile": "index.ts",
         "rootDir": "scopes/ui-foundation/uis/corner"
     },
+    "ui/debbie": {
```
Comment (davidfirst) [inline .bitmap:634]: @debs-obrien , don't forget to remove this :)

Comment (debs-obrien) [reply .bitmap]: ooops thanks

**Review by davidfirst — APPROVED** [review]

**Comment by davidfirst** [comment]
@debs-obrien , looks good, you can merge it.

**Pull request merged**